Content.Server/Anomaly/Effects/BluespaceAnomalySystem.cs

```diff
@@ -36,7 +37,11 @@ private void OnPulse(EntityUid uid, BluespaceAnomalyComponent component, ref Ano
         var range = component.MaxShuffleRadius * args.Severity * args.PowerModifier;
         var mobs = new HashSet<Entity<MobStateComponent>>();
         _lookup.GetEntitiesInRange(xform.Coordinates, range, mobs);
-        var allEnts = new ValueList<EntityUid>(mobs.Select(m => m.Owner)) { uid };
+        // list of all entities in range with the MobStateComponent
+        // we filter out those inside a ContainerSlot
+        // otherwise borg brains get removed from their body, or PAIs from a PDA
+        var allEnts = new ValueList<EntityUid>(mobs.Select(m => m.Owner)
+            .Where(m => !_container.TryGetContainingContainer((m, xformQuery.GetComponent(m), null), out var container) || container is not ContainerSlot)) { uid };
```

Comment:
Can't you just set the lookupflag to not include contained entities.

Reply:
Except the way it's done here it's filtering out anyone inside a containerslot, AKA someone in a mech is fine, or if we make a singleton storage they're also fine, which is unintuitive and relying upon other systems remaining the same instead of being purposefully designed here.

Reply:
I use a LookupFlag now to exclude contained entities.